jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java

```diff
@@ -596,7 +601,7 @@ public static ObjectName createObjectName(String domain, String name) {
 	 */
 	public static final String BUFFERPOOL_MXBEAN_DOMAIN_TYPE = "java.nio:type=BufferPool"; //$NON-NLS-1$
 
-	/*[IF !Sidecar19-SE]*/
+/*[IF JAVA_SPEC_VERSION == 8]*/
```

Comment:
The indentation of this line and line 1032 was correct: please don't change that.

Reply:
Restored the tab.